src/openapi/streaming/connection/transport/websocket-transport.ts

```diff
@@ -340,6 +353,7 @@ class WebsocketTransport implements StreamingTransportInterface {
         socket.onmessage = null;
         socket.onclose = null;
         socket.close(socketCloseCodes.NORMAL_CLOSURE, CLOSE_REASON_DESTROY);
+        this.stopInactivityFinder();

```

Comment:
we always call destroy socket right - even when the socket has closed itself?

Reply:
yes we do call socket destroy when socket close itself
i have checked the logout, reconnect and network error scenarios